lib/build-secrets/index.ts

```diff
@@ -89,7 +89,7 @@ export function generateSecretPopulationMap(
 		secretMap[serviceName] = serviceSecret;
 	}
 
-	return _.omitBy(secretMap, v => _.isEmpty(v));
+	return _.omitBy(secretMap, v => _.isEmpty(v.files));
```

Comment:
Without this change, services without secret files would fail with the error:
```
...
[Error]   Build failed
SecretPopulationError: The command '/bin/sh -c bash populate.sh' returned a non-zero code: 1
```